backend/controllers/adminController.js

```diff
@@ -1,21 +1,32 @@
-const User = require("../models/user");
+const User = require('../models/user');
 
 //Update role of a user
 function updateRole(req, res) {
-    User.update({
-        Role: req.body.role
-    }, {
+  console.log(req.body);
+
+  if (req.body.NewRole == req.body.OldRole) {
```

Comment:
We should keep the styling convention constants for the attributes of the "req.body". In this case, req.body.NewRole should be newRole, req.body.OldRole should be oldRole. The name would have to be adjusted in the front-end in this case also.